internal/handler/command.go

```diff
@@ -176,10 +176,9 @@ func execReadCmd(device *models.Device, cmd string) (*models.Event, common.AppEr
 		return nil, common.NewServerError(msg, nil)
 	}
 
-	// push to Core Data
+	// generate event
 	event := &models.Event{Device: device.Name, Readings: readings}
 	event.Origin = time.Now().UnixNano() / int64(time.Millisecond)
-	go common.SendEvent(event)
```

Comment:
This should be in a different commit than all the const changes!

Reply:
move this line for the AutoEvent OnChange mechanism, or the event will be sent here anyway